lib/go-tc/federation_resolver.go

```diff
@@ -50,6 +51,36 @@ type FederationResolver struct {
 	TypeID      *uint      `json:"typeId,omitempty" db:"type"`
 }
 
+// [V5] FederationResolverV5 is an alias for the Federal Resolver struct response used for the latest minor version associated with api major version 5.
```

Comment:
The GoDoc comment should be gin with the name of the function/ struct that it applies to.